src/app/components/card-item/card-item.component.css

```diff
@@ -0,0 +1,13 @@
+.date {
+  position: relative;
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  color: #aaacbc;
```

Comment:
ditto

Reply:
I declared font-weight, font-style, font-family in global scope.